sklearn/ensemble/_base.py

```diff
@@ -211,7 +211,10 @@ def __init__(self, estimators):
         self.estimators = estimators
 
     def _validate_estimators(self):
-        if len(self.estimators) == 0:
+        if len(self.estimators) == 0 or not all(
+            isinstance(item, tuple) and isinstance(item[0], str)
```

Comment:
Not sure how much we care, but the previous code was working fine with lists rather than tuple and with this PR it would be an error, i.e. when using `estimators=[['name', LogisticRegression()]]` rather than `estimators=[('name', LogisticRegression())]`

I guess this is documented as a list of tuple but you never know people may be using list of lists in the wild ...

Reply:
Updated to allow list in: [`56c8cad` (#30649)](https://github.com/scikit-learn/scikit-learn/pull/30649/commits/56c8cade0ea4e692d8b006355f2a7065fd703a33)